modules/ROOT/pages/dataweave-cookbook-map-an-object.adoc

```diff
@@ -34,6 +37,32 @@ include::partial$cookbook-dw/map-an-object/inputs/payload.json[]
 include::partial$cookbook-dw/map-an-object/out.json[]
 ----
 
+== Second Example
+
+This example uses the `mapObject` function to iterate through the keys and values of the object that results from using `groupBy` on the payload. If some objects of the input payload have the same values in the `FirstName`, `LastName` and `Age` keys, the DataWeave script transforms those objects into a single row in a CSV file. The remaining values in the `Team Name` and `Role` keys for those objects are concatenated with `:` in the single CSV row.
+
+[[ex02]]
+
+.Input JSON Payload:
+[source,json,linenums]
+----
+include::partial$cookbook-dw/map-an-object-ex02/inputs/payload.json[]
+----
+In the previous input example, `Sr.No. 2` and `Sr.No. 3`, and also `Sr.No. 5` and `Sr.No. 6` are exactly the same person because they have same first name, last name and age. However, `Sr.No. 4` is different person with same first and last name but different age.
```

Comment:
Suggestion: "In the previous input example, the object pair `Sr.No. 2` and `Sr.No. 3`, as well as the object pair `Sr.No. 5` and `Sr.No. 6`, refer to the same person, as they have the same first name, last name, and age. However, `Sr.No. 4` refers to a different person, as it has the same first and last name but a different age.